x-pack/plugins/apm/public/components/shared/AgentIcon/icons/php.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export function phpIcon({
+  title,
+  titleId,
+  ...props
+}: {
+  title: string;
+  titleId: string;
+} & React.SVGProps<SVGSVGElement>) {
+  return (
```

Comment:
Try to copy this approach:
https://github.com/elastic/kibana/blob/1302bba4d470a28925612dedff63b07393847218/x-pack/plugins/data_enhanced/public/search/ui/search_session_indicator/custom_icons.tsx#L15-L31

```suggestion
export function phpIcon({
  title,
  titleId,
  ...props
}: Omit<EuiIconProps, 'type'>) {
  return (
  

  
```